extension-files/tools/zabbix_discovery_devices

```diff
@@ -243,18 +243,31 @@ for filename in glob.glob('/sys/block/*'):
         for smartarg in config["smartctl_subdevices"][device_name]:
             device = {}
             device['{#BLOCKDEVICE}'] = device_name + " " + smartarg
+            device['{#DEVNAME}'] = device_name + " " + smartarg
```

Comment:
Why do we need DEVNAME and BLOCKDEVICE with the same value?

Reply:
Ah, got it :-) https://github.com/zabbix-tooling/zabbix-agent-extensions/issues/46